contrib/grinder/scripts/utils.py

```diff
@@ -177,8 +177,8 @@ def check_position(self, logger, max_position):
 
   @classmethod
   def time(cls):
-    return time.time()
+    return int(time.time() * 1000)
 
   @classmethod
   def sleep(cls, x):
-    return time.sleep(x)
+    return time.sleep(x/1000)
```

Comment:
I wanted to keep the time functions consistent.  They both use milliseconds.  It seemed confusing to me to have one output milliseconds and the other take seconds.